.github/workflows/config.yml

```diff
@@ -0,0 +1,29 @@
+name: buildpack-google-chrome
+
+on: push
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        stack: [heroku-18, heroku-20, heroku-22]
+        redis_version: [stable, beta, unstable]
```

Comment:
Not sure if the tests are working correctly. I think this should be "channel" to match line 19

Reply:
Thank you! 4d7235c